VocaDbWeb/Views/Venue/Edit.cshtml

```diff
@@ -107,7 +108,25 @@
 		</td>
 		@Html.HiddenForKnockout(m => m.Coordinates, "value: ko.toJSON(coordinates)")
 	</tr></table>
-			
+
+	<div class="editor-label">
+		<label>
+			@Res.Country
+		</label>
+	</div>
+	<div class="editor-field">
+		@Html.DropDownListFor(m => m.RegionCode, new SelectList(new RegionCollection(RegionCollection.RegionCodes).ToDictionaryFull(VocaDb.Web.Resources.Domain.Globalization.InterfaceLanguageStrings.Other), "Key", "Value", Model.RegionCode))

```

Comment:
Probably we should consider using [Select2's dropdown](https://select2.org/dropdown) for country/region selection.

Reply:
The normal dropdown is ok for now.